examples/federation-workshop/cluster-1/sentinel/Deployment.yaml

```diff
@@ -0,0 +1,76 @@
+# /*
+# |    Protect your secrets, protect your sensitive data.
+# :    Explore VMware Secrets Manager docs at https://vsecm.com/
+# </
+# <>/  keep your secrets… secret
+# >/
+# <>/' Copyright 2023–present VMware, Inc.
+# >/'  SPDX-License-Identifier: BSD-2-Clause
+# */
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: vsecm-sentinel
+  namespace: vsecm-system
+  labels:
+    app.kubernetes.io/name: vsecm-sentinel
+    app.kubernetes.io/part-of: vsecm-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: vsecm-sentinel
+      app.kubernetes.io/part-of: vsecm-system
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: vsecm-sentinel
+        app.kubernetes.io/part-of: vsecm-system
+    spec:
+      serviceAccountName: vsecm-sentinel
+      containers:
+        - name: main
+          image: localhost:32000/vsecm-ist-sentinel:0.21.2
+          volumeMounts:
+            - name: spire-agent-socket
+              mountPath: /spire-agent-socket
+              readOnly: true
+          #
+          # You can configure VSecM Sentinel by providing environment variables.
+          #
+          # See https://vsecm.com/configuration for more information about
+          # these environment variables.
+          #
+          # When you don’t explicitly provide env vars here, VMware Secrets Manager
+          # Sentinel will assume the default values outlined in the given link above.
+          #
+          env:
+            - name: SPIFFE_ENDPOINT_SOCKET
+              value: "unix:///spire-agent-socket/agent.sock"
+            - name: VSECM_WORKLOAD_SVID_PREFIX
+              value: "spiffe://cluster1.demo/workload/"
+            - name: VSECM_SENTINEL_SVID_PREFIX
+              value: "spiffe://cluster1.demo/workload/vsecm-sentinel/ns/vsecm-system/sa/vsecm-sentinel/n/"
+            - name: VSECM_SAFE_SVID_PREFIX
+              value: "spiffe://cluster2.demo/workload/vsecm-safe/ns/vsecm-system/sa/vsecm-safe/n/"
+            - name: VSECM_PROBE_LIVENESS_PORT
+              value: ":8081"
+            - name: VSECM_SAFE_ENDPOINT_URL
+              value: "https://10.0.0.71:8443"
+          livenessProbe:
+            httpGet:
+              path: /
+              port: 8081
+            initialDelaySeconds: 1
+            periodSeconds: 10
+      volumes:
+        # Using SPIFFE CSI Driver to bind to the SPIRE Agent Socket
+        # ref: https://github.com/spiffe/spiffe-csi
+        - name: spire-agent-socket
+#          csi:
+#            driver: "csi.spiffe.io"
+#            readOnly: true
+          hostPath:
```

Comment:
I had to use hostpath because microk8s did not like the CSI driver. In a regular cluster, we can switch to csi driver.